<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KgvPageLayoutColumnsBase</class>
 <widget class="QWidget" name="KgvPageLayoutColumnsBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>169</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <item row="1" column="1">
    <widget class="QFrame" name="columnSpacingPane">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>Columns:</string>
     </property>
     <property name="buddy">
      <cstring>nColumns</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelSpacing">
     <property name="text">
      <string>Column spacing:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="QFrame" name="previewPane">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="nColumns">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>16</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
